scripts/bundle-rhmi-operators.sh

```diff
@@ -28,17 +26,20 @@ UPGRADE_RHMI="${UPGRADE:-false}"
 VERSIONS="${BUNDLE_VERSIONS:-$LATEST_VERSION}"
 ROOT=$(pwd)
 INDEX_IMAGE=""
+CATALOGUE_SOURCE_INSTALL="${OC_INSTALL:-true}"
 
 
 start() {
-  # TODO: validate input
+  clean_up
   create_work_area
   copy_bundles
   check_upgrade_install
   generate_bundles
   generate_index
+  if [ "$CATALOGUE_SOURCE_INSTALL" = true ] ; then
   create_catalog_source
-  clean_up
+  fi
+  #clean_up
```

Comment:
```suggestion
  clean_up
  fi
```

I like the approach of cleaning up in the first place in case the temporal folder was still there (very useful to look into it for debugging purposes), but I'd keep the clean up at the end in order to keep git clean. Other approach would be adding the temp folder to the `.gitignore`

Reply:
ah good spot, I've commented it out just to see how do the bundles look like, forgot to uncomment it! thank you!